server/parsec/components/account.py

```diff
@@ -407,6 +413,59 @@ def _convert_vault(
             case AccountVaultItemRecoveryList.ACCOUNT_NOT_FOUND:
                 client_ctx.account_not_found_abort()
 
+    async def _send_email_deletion_token(
+        self, token: AccountDeletionToken, email: EmailAddress
+    ) -> None | SendEmailBadOutcome:
+        if not self._config.server_addr:
+            return SendEmailBadOutcome.BAD_SMTP_CONFIG
+
+        deletion_url = ParsecAccountDeletionAddr.build(
+            server_addr=self._config.server_addr,
+            token=token,
+        ).to_http_redirection_url()
+
+        message = generate_email_deletion_email(
+            from_addr=self._config.email_config.sender,
+            to_addr=email,
+            deletion_url=deletion_url,
+            server_url=self._config.server_addr.to_http_url(),
+        )
+
+        return await send_email(
+            email_config=self._config.email_config, to_addr=email, message=message
+        )
+
+    async def create_email_deletion_token(
+        self,
+        email: EmailAddress,
+        now: DateTime,
+    ) -> AccountDeletionToken | AccountCreateAccountDeletionTokenBadOutcome:
+        raise NotImplementedError
+
+    @api
+    async def api_account_delete_send_validation_email(
+        self,
+        client_ctx: AuthenticatedAccountClientContext,
+        req: authenticated_account_cmds.latest.account_delete_send_validation_token.Req,
+    ) -> authenticated_account_cmds.latest.account_delete_send_validation_token.Rep:
+        outcome = await self.create_email_deletion_token(client_ctx.account_email, DateTime.now())
+        match outcome:
+            case AccountDeletionToken() as token:
+                outcome = await self._send_email_deletion_token(token, client_ctx.account_email)
+                match outcome:
+                    case None:
+                        return authenticated_account_cmds.latest.account_delete_send_validation_token.RepOk()
+                    case (
+                        SendEmailBadOutcome.BAD_SMTP_CONFIG | SendEmailBadOutcome.SERVER_UNAVAILABLE
+                    ):
+                        return authenticated_account_cmds.latest.account_delete_send_validation_token.RepEmailServerUnavailable()
+                    case SendEmailBadOutcome.RECIPIENT_REFUSED:
+                        return authenticated_account_cmds.latest.account_delete_send_validation_token.RepEmailRecipientRefused()
+            case AccountCreateAccountDeletionTokenBadOutcome.TOO_SOON_AFTER_PREVIOUS_DEMAND:
+                return (
```

Comment:
I'd add same comment we used for create account
```suggestion
                # Respond OK without sending token to prevent creating oracle
                return (
```

Reply:
1. This is an authenticated route that only use the email of the authenticated user, at that point you already know the email of the user.
2. _Too soon after previous demand_ is not about creating an oracle but preventing an user to spam emails